### DISCUSSION TITLE
Updating all products with 'delivery': 'shipment'

### DISCUSSION THREAD

**Discussion by Kealtie** [swellstores/community#519] [Help] [answered]
Some of our Products in Swell might not have 'delivery': 'shipment' resulting in no shipment costs being added to the cart. 

How would I give all products 'delivery': 'shipment' through the Console?

**Comment by logeshswell** [comment, marked as answer]
You need to set up the product.type = 'standard' (Physical product)
For more info, please check: https://developers.swell.is/backend-api/products/the-product-model
